miqa/core/migrations/0018_image_to_frame.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 3.2.8 on 2021-11-05 19:29
+import uuid
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0017_remove_site'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
I think you may have to hand-edit this migration to make it use a `migrations.RenameModel`. As written, this will delete all existing images in the system.

Reply:
See https://stackoverflow.com/a/26241640/2550451

Reply:
Good idea. I've been wondering for a while, though, if there's ever a good time to squash all migrations to fewer files, since a lot of our current migration files go back and forth. Is that a good idea? Or do we want to keep all of these previous migration generations?

Reply:
I hadn't thought of that. Adding change to next commit.